docs/versions/latest/Components/Archetypes/typescript/Components/Policy.md

```diff
@@ -0,0 +1,309 @@
+# Policies
+vRO Policies and how to define them in Aria Orchestrator
+
+## Overview
+Aria Orchestrator supports creating Policies with different elements and variables.
+
+## Table of Contents
+1. [Policy Template Versioning](#policy-template-versioning)
+2. [Example Policy](#exmaple-policy)
+3. [Naming Convention](#naming-convention)
+4. [Creating Variables in Policy](#creating-a-variable-in-policy)
+5. [Creating Elements to a Policy](#creating-a-elements-to-a-policy)
+
+## Policy Template Versioning
+Policy template now has two versions for backward compatability. Template versions can be changed using the **templateVersion** attribute. Use v2 for full support of creating Policy with variables and multiple elements.
+
+**NOTE:** templateVersion attribute is optional. if not provided, templateVersion will be considered as v1 by default.
```

Comment:
I don't see this being used in any way, we've defined it but nothing else, how are we validating this?

Reply:
Policy Template Versioning was introduced to make the new changes be backward compatible. If user provides version as v2 which is optional, user can create policy with full support.